docs/runtime/core.md

```diff
@@ -7,72 +7,72 @@ Runtime Core is a compilation/execution engine for neural network models.
 
-There is a scheduler that allows the user manually specify backends via compiler options. There is another scheduler that automatically assigns backends based on profile information measured in advance and saved.
+There is a scheduler that allows the user to manually specify backends via compiler options. There is another scheduler that automatically assigns backends based on profile information measured in advance and saved.
 
 #### 2. Tensor Registration
 
-Each backend manages its tensors. In this phase operand informations get registered to the corresponding backend. This will be used generating tensor objects.
+Each backend manages its tensors. In this phase, operand informations get registered to the corresponding backend. This will be used in generating tensor objects.
 
-##### Q. What are differences between 'operand' and 'tensor'?
+##### Q. What are the differences between 'operand' and 'tensor'?
 
 In **ONE** runtime, 'operand' refers to an operand in a neural network model. While 'tensor' includes all 'operand' info plus actual execution info like actual buffer pointer. In short, 'operand' is for `ir`, 'tensor' is for `backend`.
 
 #### 3. Linearization (Linear Executor Only)
 
-Linearization means sorting operations in a topological order. It saves execution time since it is not needed to resolve next available operations after every operation at execution time.
+Linearization means sorting operations in topological order. It saves execution time since it is not needed to resolve the next available operations after every operation at execution time.
 
 It also makes plans for tensor memory. It can save some memory space by reusing other operands' space that does not overlap lifetime. All allocations are done at compile time (after [4. Kernel Generation](#4.-kernel-generation)) which saves execution time too.
 
 #### 4. Kernel Generation
 
-'kernel' here means an implementation of actual calculation of an operation.
+'kernel' here means an implementation of the actual calculation of an operation.
 
 A backend is assigned for each operation. In this phase, a kernel for each operation is generated.
 
 Let's say we have some functions written in a certain programming language. Then its compiler compiles each function into a chunk of assembly. Here 'function' is like 'operation' and 'assembly' is like 'kernel'.
 
 #### 5. Create Executor
 
-With generated tensors and kernels, the compiler creates executor objects. There are 3 types of executors are supported - Linear, Dataflow and Parallel. Linear executor is the default executor and Dataflow Executor and Parallel Executor are experimental.
+With generated tensors and kernels, the compiler creates executor objects. There are 3 types of executors are supported - Linear, Dataflow, and Parallel. Linear executor is the default executor and Dataflow Executor and Parallel Executor are experimental.
```

Comment:
Why did you add `,` after `Dataflow`?

Reply:
Hmm.. I also thought it was strange but Grammarly made this change 🙄